src/dataframe/dataframe.jl

```diff
@@ -763,6 +763,45 @@ end
 ##
 ##############################################################################
 
+"""
+    deletecols!(df::DataFrame, ind)
+
+Delete columns from a data frame in place.
+
+
+### Arguments
+
+* `df` : the DataFrame from which we want to remove a column
+
+* `ind` : a position from which we want to remove a column
```

Comment:
"position" makes it sound like it can only be an integer, but it can also be a symbol (the signature should say so).

BTW, I think we should stop having separate Arguments and Result sections when the function is simple (as recommended in the Julia manual). Here a simple sentence or two would be as clear.

Reply:
fixed. I also a fixed a bug in `deletecols!` as it allowed duplicate column identifier to be passed.